add-on/chrome/test/mochitest/browser_throttler.js

```diff
@@ -0,0 +1,167 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+
```

Comment:
Throttler is created in bootstrap.js and not a service, so it would be tricky to test via xpcshell.